documentation/guide-batch-layer.asciidoc

```diff
@@ -7,791 +7,215 @@ We understand batch processing as bulk-oriented, non-interactive, typically long
 
 devonfw uses link:http://projects.spring.io/spring-batch/[Spring Batch] as batch framework. 
 
-This guide explains how Spring Batch is used in devonfw applications. Please note that it is not yet fully consistent concerning batches with the sample application. You should adhere to this guide by now.
+This guide explains how Spring Batch is used in devonfw applications. It focuses on aspects which are special to devonfw if you want to learn about spring-batch you should adhere to springs references documentation.
+
+There is an example of simple batch implementation in the https://github.com/devonfw/my-thai-star/tree/develop/java/mtsj/batch[my-thai-star batch module]. 
 
-== Batch architecture
 
 In this chapter we will describe the overall architecture (especially concerning layering) and how to administer batches. 
 
-=== Layering
+== Layering
 
 Batches are implemented in the batch layer. The batch layer is responsible for batch processes, whereas the business logic is implemented in the logic layer. Compared to the link:guide-service-layer.asciidoc[service layer] you may understand the batch layer just as a different way of accessing the business logic.
 From a component point of view each batch is implemented as a subcomponent in the corresponding business component.
 The business component is defined by the link:architecture.asciidoc[business architecture].
 
-Let's make an example for that. The sample application implements a batch for exporting bills. This _billExport_ belongs to the salesmanagement business component.
-So the _billExport_ is implemented in the following package:
+Let's make an example for that. The sample application implements a batch for exporting ingredients. This ingredientExportJob belongs to the dishmanagement business component.
+So the ingredientExportJob is implemented in the following package:
 //Example doesn't exist anymore and I didn't find any other used batches. 
 [source]
-<basepackage>.salesmanagement.batch.impl.billexport.*
+<basepackage>.dishmanagement.batch.impl.*
 
 Batches should invoke use cases in the logic layer for doing their work. 
 Only "batch specific" technical aspects should be implemented in the batch layer.
 
 ==========================
 Example:
-For a batch, which imports product data from a CSV file, this means that all code for actually reading and parsing the CSV input file is implemented in the batch layer.
+For a batch, which imports product data to a CSV file, this means that all code for actually reading and parsing the CSV input file is implemented in the batch layer.
```

Comment:
`from` as actually correct for an import. Did you mean export?

Reply:
You're like. I did not change this intentionally. Maybe some weird merging issue....